lib/tweetflood/twitter_stream.ex

```diff
@@ -88,8 +88,9 @@ defmodule Tweetflood.TwitterStream do
   end
 
   @doc false
```

Comment:
We don't need `@doc false` on private functions, hex already ignores `@doc` for private functions so this does nothing.